docs/components/networktransform.md

```diff
@@ -121,93 +225,113 @@ darkImageSrc="/BufferedTick_Dark.png?text=DarkMode"/>
   <figcaption>Graphic of a buffered tick between the server and a client (that is, interpolation)</figcaption>
 </figure>
 
-### Slerp position
 
-When this property and **Interpolation** are both set, non-authoritative instances will [Slerp](https://docs.unity3d.com/ScriptReference/Vector3.Slerp.html) towards their destination position as opposed to [Lerp](https://docs.unity3d.com/ScriptReference/Vector3.Lerp.html). Typically this can be used when your object is following a circular and/or spline based motion path and you want to preserve the curvature of that path. Since "lerping" between two points yields a linear progression over a line between two points, there can be scenarios where the frequency of delta position state updates could yield a loss in the curvature of an object's motion.  
+#### Legacy Lerp Interpolator Type
```

Comment:
Should we really have the legacy lerp type still be the default? Is there a reason for that, or is it just a backwards-compatibility thing?